index.js

```diff
@@ -40,15 +41,6 @@ app.use(function(req, res, next) {
   next();
 });
 
-app.get("/", function(req, res) {
-  res.json({
-    name: pkgInfo.name,
-    documentation: "https://github.com/n1k0/readable-proxy/blob/master/README.md",
-    description: pkgInfo.description,
-    version: pkgInfo.version
-  });
-});
```

Comment:
(@ametaireau speaking). I like the fact to have this / endpoint with the name, doc, desc and version.